### PR TITLE
Created the UI for Brand pg

### LINKED ISSUE #1692: Meshery Brand Page Update [Website]
#### Current State of the Meshery.io site
The current mehsery brand page meshery.io/brand does not meet up to code requirements and needs to be restructured. An update has also been made to the design which needs to be included in the current implementation.

#### Desired State of the Meshery.io site
Restructure the code for the Meshery Brand page and ensure it meets up to the desired requirements. Link to [brand page](https://meshery.io/brand).
Include the updates to the designs in Figma. Find the designs in the Meshery UI design file using [this link](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=27509%3A126093&mode=design&t=TQ3giaDQe33TH1HI-1).

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by psankhe28**
**Description**

This PR fixes #1692 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

Video:

[brand-pg](https://github.com/meshery/meshery.io/assets/84843461/cbf40625-57bb-4ce6-83ca-e21ede0b4096)

### CODE REVIEW

**Comment by psankhe28** [comment]
@Rexford74, can you please review this?

**Comment by Rexford74** [comment]
Please be sure to drop in a message on slack when you've updated the PR so that we can review this and get it merged. Well done, Pratiksha.

**Review by Rexford74 — CHANGES_REQUESTED** [review]
Let me know when you have some time, Pratiksha. These changes can be rounded up and the PR merged before tomorrow's end. Just a few changes left. Please let me know what you decide. Thank you.

**Comment by ritiksaxena124** [comment]
Merge conflicts @psankhe28

**Comment by leecalcote** [comment]
@Rexford74 ready for review.

**Comment by Rexford74** [comment]
Hey, Pratiksha. Thank you your efforts on this so far. Just a little adjustments left to get this merged please if you will. I have highlighted the needed changes in a video for better understanding. Please let me know if this is clear or if you will require further clarification.
Thank you.
[dc03465b-f852-4c40-8544-bea031873c5d.webm](https://github.com/meshery/meshery.io/assets/98197344/7563db78-dfc6-4c85-bb0a-11716a594675)

**Comment by psankhe28** [comment]
> Hey, Pratiksha. Thank you your efforts on this so far. Just a little adjustments left to get this merged please if you will. I have highlighted the needed changes in a video for better understanding. Please let me know if this is clear or if you will require further clarification. Thank you. [Video Link](https://github.com/meshery/meshery.io/assets/98197344/7563db78-dfc6-4c85-bb0a-11716a594675)

The space is due to the layout which is used as a base for all the pages.

File: _sass/brand.scss [hunk 1]
```diff
@@ -168,15 +181,19 @@
     width: 100%;
     display: flex;
     color: white;
-    gap: 1rem; /* space between color boxes */
-    margin-top: 2rem; /* space above the color palette */
-    flex-wrap: wrap; /* Allow color boxes to wrap */
-    justify-content: center; /* Distribute items evenly */
+    gap: 1rem;
```
Comment (vishalvivekm) [inline _sass/brand.scss:184]: Please keep the comments as is i.e. to the right of the declarations.

File: _sass/brand.scss [hunk 2]
```diff
@@ -256,45 +347,93 @@
...
+        justify-content: center;
+        align-items: center;
+        flex-direction: row;
+        margin: 0;
+        padding: 0;
+    }
+
+    .color-box {
+        width: 140px;
+        height: 140px;
```
Comment (vishalvivekm) [inline _sass/brand.scss:403]: Let's please avoid writing "px" to the extent possible @psankhe28.
can be replaced by "rem", "em" etc.

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
left some comments..

File: _sass/brand.scss (hunk 2 above)
Comment (psankhe28) [reply _sass/brand.scss:403]: It was discussed with Rex and he suggested to use 140px

File: collections/_pages/brand.html [hunk 3]
```diff
@@ -390,6 +494,8 @@ <h1 class="layerH3" style="font-weight: 500">Our Mascot, Five</h1>
                     >Presentation Slides</a
                 >
             <a href="#social" class="header-link">Social Posts</a>
-            <!-- <a href="#mascot" class="header-link">Our Mascot, Five</a> -->
+            <a href="#mesh-sync" class="header-link">MeshSync</a>
+            <a href="#meshsync-operator" class="header-link">MeshSync Operators</a>
```
Comment (ritiksaxena124) [inline collections/_pages/brand.html:498]:
```suggestion
            <a href="#meshsync-operator" class="header-link">MeshSync Operator</a>
```

File: collections/_pages/brand.html [hunk 4]
```diff
@@ -318,70 +295,197 @@ <h1>Social Posts</h1>
...
+                    class="logo-container-img1"
+                />
+                <img
+                    src="{{ site.baseurl }}/assets/images/brand/meshsync-logo-1.svg"
+                    class="meshsync-logo-container-img"
+                />
+            </div>
+            <h2>Colors</h2>
+            <p>
+                The Meshery color palette consist of the primary Meshery color and additional shades. The Meshery logo should be white or monochrome tonal when using a color background. Visit Sistent Design System at  <a class='brand-links' href"https://docs.google.com/presentation/d/1Ofy15e2bAIz4IYC8X70omnlpX9vI0QZpkRijYbxGAd4/edit#slide=id.g232ed436c58_0_144">design.layer5.io</a> for more guidance on how to apply Mehsery colors.
```
Comment (ritiksaxena124) [inline collections/_pages/brand.html:317]:
```suggestion
                The Meshery color palette consist of the primary Meshery color and additional shades. The Meshery logo should be white or monochrome tonal when using a color background. Visit <a class='brand-links' href"https://layer5.io/projects/sistent">Sistent Design System</a> for more guidance on how to apply Mehsery colors.
```

**Comment by sudhanshutech** [comment]
@psankhe28 will you share your progress in website meet if you don't have exam

**Comment by psankhe28** [comment]
> @psankhe28 will you share your progress in website meet if you don't have exam

I have already shown it twice and in the last meet it was decided to merge but it is not merged yet.
I have my exams till 10 June so I won't be able to join.

**Pull request closed without merging**